save-backend/src/main/kotlin/org/cqfn/save/backend/repository/TestExecutionRepository.kt

```diff
@@ -1,10 +1,20 @@
 package org.cqfn.save.backend.repository
 
+import org.cqfn.save.domain.TestResultStatus
 import org.cqfn.save.entities.TestExecution
+import org.springframework.data.domain.Pageable
 import org.springframework.stereotype.Repository
 
 /**
  * Repository of execution
  */
 @Repository
-interface TestExecutionRepository : BaseEntityRepository<TestExecution>
+interface TestExecutionRepository : BaseEntityRepository<TestExecution> {
+    /**
+     * @param status
+     * @param id
+     * @param pageable
+     * @return list of test executions
+     */
+    fun findAllByStatusEqualsAndTestSuiteExecutionIdEquals(status: TestResultStatus, id: Long, pageable: Pageable): List<TestExecution>
```

Comment:
I think, `equals` is the default behavior for spring-data-jpa, so it can be omitted. Also, `findBy...` with return type of collection should be equivalent to `findAllBy...`